src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export function Person({ person }) {}
+import React from 'react';
+
+export function Person({ person }) {
+  const partnerString = person.sex === 'm' ? (
+    `My wife's name is ${person.partnerName}`
```

Comment:
Use destructuring for getting access to values of props object https://github.com/mate-academy/react_person/blob/master/checklist.md